helpers/compile/build.ts

```diff
@@ -123,7 +123,7 @@ async function dependencyCheck(options: BuildOptions) {
   // we need to bundle everything to do the analysis
   const buildPromise = esbuild.build({
     entryPoints: glob.sync('**/*.{j,t}s', {
-      ignore: ['./src/__tests__/**/*'],
+      ignore: ['./src/__tests__/**/*', './tests/ecosystem/**/*'],
```

Comment:
We don't check dependencies in ecosystem tests because tests are isolated from the build.